src/optimizer/statistics/selectivity_util.cpp

```diff
@@ -71,6 +125,10 @@ double SelectivityUtil::Equal(common::ManagedPointer<NewColumnStats<T>> column_s
 
   size_t numrows = column_stats->GetNumRows();
 
+  if (numrows == 0) {
+    return 0;
```

Comment:
I'm not actually sure if this should be 0 or 1. What's the selectivity of a column with no rows?